pkg/dashboard/static/styles.css

```diff
@@ -141,9 +141,9 @@ body > .container-fluid {
     font-size: 0.75rem;
 }
 
-#installedList .body .row div div {
+/* #installedList .body .row div div {
```

Comment:
If it's not needed - just remove it. GitHub history will remember it.

Reply:
commented by mistake. Reverted it